test/eChronicler_web/controllers/page_controller_test.exs

```diff
@@ -3,6 +3,13 @@ defmodule EChroniclerWeb.PageControllerTest do
 
   test "GET /", %{conn: conn} do
     conn = get(conn, "/")
-    assert html_response(conn, 200) =~ "Welcome to E-Chronicler!"
+    assert html_response(conn, 200) =~ "Journal Entries"
   end
+
+  test "Journal Entries are Displayed at /", %{conn: conn} do
```

Comment:
Not quite hitting the mark, but we will discuss.  Once you get test up for list_journal_entries, use identical set up to create, say, three journal entries.  Then test, for example, that each title is in the html.